doc/2/guides/getting-started/store-and-access-data/index.md

```diff
@@ -105,7 +105,7 @@ Select the `nyc-open-data` index and then the `yellow-taxi` collection. You shou
 
 ### Search for documents
 
-Kuzzle directly exposes [Elasticsearch's query language](https://www.elastic.co/guide/en/elasticsearch/reference/7.4/query-dsl.html) in a secure way. 
+Kuzzle directly exposes directly the data through the API by allowing client to express a query to match documents they need. 
```

Comment:
I don't quite get what you're trying to say here, so the following suggestion might not be spot on :thinking: 

```suggestion
Kuzzle allows to search for documents through its API, with different query languages depending on your needs.
```